src/lifetracker/calendar/CalendarList.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.time.*;
```

Comment:
Try to use individual imports. Eclipse can be set to do it automatically in "Preferences -> Java -> Code Style -> Organize Imports". 

Set "Number of imports need for *" in both fields to some large number like 99.

Then do Shift-Ctrl-O to organize imports.